freebsd build of 1.5.0 fails on fileutils: syncDir redeclared

[immudb15-1.5.0.log](https://github.com/codenotary/immudb/files/11834120/immudb15-1.5.0.log)

build failure:
[00:03:02] github.com/codenotary/immudb/embedded/appendable/fileutils
[00:03:02] # github.com/codenotary/immudb/embedded/appendable/fileutils
[00:03:02] embedded/appendable/fileutils/fileutils_unix_nonlinux.go:24:6: syncDir redeclared in this block
[00:03:02] 	embedded/appendable/fileutils/fileutils_freebsd.go:25:6: other declaration of syncDir
[00:03:02] embedded/appendable/fileutils/fileutils_unix_nonlinux.go:35:6: fdatasync redeclared in this block
[00:03:02] 	embedded/appendable/fileutils/fileutils_freebsd.go:36:6: other declaration of fdatasync

Happened on FreeBSD 13.2 on aarch64.

--- a/embedded/appendable/fileutils/fileutils_unix_nonlinux.go
+++ b/embedded/appendable/fileutils/fileutils_unix_nonlinux.go
@@ -1,5 +1,5 @@
-//go:build unix && !linux && !darwin
-// +build unix,!linux,!darwin
+//go:build unix && !linux && !darwin && !freebsd
+// +build unix,!linux,!darwin,!freebsd
 
 /*
 Copyright 2023 Codenotary Inc. All rights reserved.

